src/sentry/eventstore/models.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import absolute_import
+
+from datetime import datetime
+from django.utils import timezone
+
+from sentry.models import EventCommon, EventDict
+from sentry.db.models import NodeData
+
+
+def ref_func(x):
+    return x.project_id or x.project.id
+
+
+class Event(EventCommon):
+    def __init__(self, project_id, event_id, group_id=None, data=None):
+        self.project_id = project_id
+        self.event_id = event_id
+        self.group_id = group_id
+        self.data = data
```

Comment:
The django model also had:
- message
- platform
- datetime
- time_spent
as public fields.
I think you deprecated message. But what bout time_spent ? Is there anyone accessing it ?

Reply:
Message is not deprecated. It just gets set here https://github.com/getsentry/sentry/blob/master/src/sentry/event_manager.py#L613 but this will need to move into our Event class in future.

time_spent is not used anywhere (final bits of cleanup were done in https://github.com/getsentry/sentry/pull/15701)